INSTALL.md

```diff
@@ -284,81 +284,84 @@ If there are other files that you have customized, then you will also need to tr
+ ```
+ max_execution_time = 600
+ max_input_time = 600
+ max_input_vars = 3000
+ memory_limit = 512M
+ post_max_size = 32M
+ upload_max_filesize = 32M
+ session.gc_maxlifetime = 14400
+ short_open_tag = On
+ display_errors = Off
+ upload_tmp_dir is set to a correct default value that will work on your system
+ error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_NOTICE
+ ```
+ **Remember to save your file, so that your settings don't reset!**
+
+ **Make sure that strict mode is disabled in Mysql**
+ 
+ **[How to disable Mysql strict mode?](#how-to-disable-mysql-strict-mode-?)**
+ 
+ #### Step 1
+Launch XAMPP Control Panel and navigate to LibreEHR Setup: `\LibreEHR\setup.php`
+ 
+ **Note:**
+ 1. Make sure that your XAMPP control panel has Apache and MySQL turned on
+ 2. Apache needs to be on port 80, 443 (Try some of the following suggestions on [this forum](https://stackoverflow.com/questions/8103924/xampp-port-80-is-busy-easyphp-error-in-apache-configuration-file) if you are experiencing issues or check out the [FAQs](#faq) on how to contact us)
+ 3. mySQL needs to be on port 3306
+ 
+ #### Step 2
+ Leave default as the "Site ID:" and press continue.
+ 
+ ![First Step](./Documentation/1_Installing/images/windows_installation/Step_1.png)
+ 
+ Make sure that there are no undefined index errors, if so make sure that you changed the php.ini file, or have the correct version of XAMPP.
+ 
+ ![Second Step](./Documentation/1_Installing/images/windows_installation/Step_2.png)
+ 
+ Then after the second step, continue and leave the option "Have setup create the database" and press continue
+ 
+ ![Third Step](./Documentation/1_Installing/images/windows_installation/Step_3.png)
+ 
+ For the fourth step, enter a "Password" and "Initial User Password" you are free to change, the "Initial User" to your own username, but for convenience you can also leave it as admin. Then press continue.
+**Note:** If you do change your password, or even if you don't, write down your username and password. You will need it for future uses.
+ 
+ ![Fourth Step](./Documentation/1_Installing/images/windows_installation/Step_4.png)
+ 
+ Then after the fourth step, you can press continue through the others as long as the steps above were followed with clarity, the rest should have no errors, and each remaining page can be continued without change.
+ 
+ ![Fifth Step](./Documentation/1_Installing/images/windows_installation/Step_5.png)
+ 
+ ![Sixth Step](./Documentation/1_Installing/images/windows_installation/Step_6.png)
+ 
+ ![Seventh Step](./Documentation/1_Installing/images/windows_installation/Step_7.png)
+ 
+ ![Eigth Step](./Documentation/1_Installing/images/windows_installation/Step_8.png)
+ 
+ ![Ninth Step](./Documentation/1_Installing/images/windows_installation/Step_9.png)
+ 
+ ![Tenth Step](./Documentation/1_Installing/images/windows_installation/Step_10.png)
+ 
+ **Once you see the screen above, you have successfull setup LibreEHR! Congratulations!**
```

Comment:
successfull should be successful , would it sound better as successfully

Reply:
Sorry didn't notice that typo.

Reply:
Man I am lucky if I don't mess them up big time. Liked you bowtie